components/global/CtBetaAlert.vue

```diff
@@ -0,0 +1,10 @@
+<script>
+export default {}
+</script>
+
+<template>
+  <Alert type="warning">
+    <Icon name="exclamation-triangle"></Icon> Component Testing is currently in
+    beta.
```

Comment:
This seems to need surrounding `<p></p>` tags to render correctly.

```suggestion
    <p><Icon name="exclamation-triangle"></Icon> Component Testing is currently in
    beta.</p>
```

Before

![image](https://user-images.githubusercontent.com/54051/163030987-8a289732-95a8-4ce4-9551-73de89082f3e.png)

After

![image](https://user-images.githubusercontent.com/54051/163031004-77ee8247-e92a-4596-afa9-48a1535750e7.png)